Tools/TestWebKitAPI/Tests/WTF/SequenceLockedTest.cpp

```diff
@@ -66,7 +66,7 @@ TEST_F(SequenceLockedTest, Works)
         thread->waitForCompletion();
     });
     unsigned maxValue = 0;
-    for (int i = 0; i < 100000; ++i) {
+    for (int i = 0; i < 10000; ++i) {
```

Comment:
could we have something like
```
#ifndef NDEBUG
constexpr int iterations = 1000;
#else
constexpr int iterations = 10000;
#endif
```

There was the problem that there was a  bug that didn't reproduce with 1000 iterations (on release).

Reply:
Sure! I had done this based on your comment that "the bug didn't reproduce with 1000 iteration"